examples/credential-registry-storage-contract/src/lib.rs

```diff
@@ -236,7 +223,7 @@ pub struct SerializationHelperParam {
 #[receive(
```

Comment:
Please update the comment which talks about `SerilaizationHelperParam`.

Reply:
Fixed now. Good catch.